SPECS/python-optree/python-optree.spec

```diff
@@ -30,18 +32,24 @@ A PyTree is a recursive structure that can be an arbitrarily nested Python conta
 
 
 %build
+# Remove -D_GLIBCXX_ASSERTIONS because optree throws std lib assertion errors.
```

Comment:
Sounds a bit like we're burying a legitimate error, which is not really what's happening. How about this:
```suggestion
# Remove "-D_GLIBCXX_ASSERTIONS" because optree will otherwise crash
# on an C++ std libs assertion error. This happens when one tries to
# call 'map_tree' on an empty tuple (or possibly all empty collections).
# We currently suspect an upstream bug, as 'map_tree' seems to work
# fine when compiled without the "_GLIBCXX_ASSERTIONS" macro.
```